spec/assignment_helper_spec.rb

```diff
@@ -23,5 +23,21 @@
         expect { stub_test_track_assignments(foo: :bar) }.to raise_error(/Cannot stub test track assignments/)
       end
     end
+
+    it 'works correctly with a vary call' do
```

Comment:
I added a more integration-y test for `stub_test_track_assignments` that would have caught this issue sooner.